tests/storetests.js

```diff
@@ -0,0 +1,235 @@
+//store.js tests
+
+QUnit.module("LocalStorage Store Tests", {
+  beforeEach: function() {
+    // prepare something before each test
+    this.lstore = new LocalCalendarStorage({storeId: 'test'});
+  },
+  afterEach: function() {
+    // clean up after each test
+  }
+});
+
+QUnit.test("initLocalStorage test", function(assert) {
+    assert.expect(2);
+    assert.equal(this.lstore.activeCalls, 0);
+    assert.deepEqual(this.lstore.activityChangeFunctions, []);
+});
+
+QUnit.test("toKey Test", function(assert) {
+    assert.expect(1);
+    var id = '1234';
+    var prefix = 'test';
+    var keyId = this.lstore._toKey(id);
+    assert.equal(keyId, 'test_1234');
+});
+
+QUnit.module("localCalendarStorage and checkit Tests", {
+    beforeEach: function() {
+        // Clear localStorage before putting anything in it
+        localStorage.clear();
+        this.$fixture = $('#qunit-fixture');
+        this.$calendarDiv = this.$fixture.find('#calendarDiv');
+        
+        this.$fixture.append(
+            `<script>
+            $(document).ready( function() {
+            checkit = new CheckIt('localStorage');
+            });
+            </script>`);
+        this.checkit = checkit;
```

Comment:
I don't think it's the same checkit. I need to use checkit functions that's why I did line 40.